server/jetstream_cluster_test.go

```diff
@@ -10677,19 +10677,30 @@ func TestJetStreamClusterStreamReplicaUpdateFunctionCheck(t *testing.T) {
 
 	// Make sure cluster replicas are current.
 	for _, r := range si.Cluster.Replicas {
-		if !r.Current {
-			t.Fatalf("Expected replica to be current: %+v", r)
-		}
+		checkFor(t, 5*time.Second, 100*time.Millisecond, func() error {
+			if !r.Current {
```

Comment:
Oh, that's for nothing, the state of si.Cluster.Replicas is not going to change, so r.Current would always be the same. It would have to include the StreamInfo request itself. Since I have not been able to have it flap either, you can remove for now.

Reply:
Fixed